browser/brave_profile_prefs.cc

```diff
@@ -177,6 +177,9 @@ void RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry) {
   registry->RegisterStringPref(kBraveWalletEncryptedSeed, "");
   registry->RegisterBooleanPref(kBraveWalletEnabled, true);
 
+  // Autocomplete in address bar
+  registry->RegisterBooleanPref(kAutocompleteEnabledInAddressBar, true);
```

Comment:
please drop `InAddressBar` from all the code. I don't care what we display to the users, but in the code is should just be simple autocomplete enabled or not

Reply:
Fixed with https://github.com/brave/brave-core/pull/4033/commits/e35af85a63d3821d278519cb3cef9e0f08f48af1